packages/admin/admin-icons/generate-icons.ts

```diff
@@ -84,6 +84,7 @@ const getSVGData = (icon: Icon) => {
 };
 
 const writeComponent = async (icon: Icon, svgString: string) => {
+    const searchWords = icon.name.replace(/\.[^/.]+$/, "");
```

Comment:
Instead of "words" I'd use "terms", which is more common in search, what do you think?